Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if USE_IMPORT_FOUNDATION
```

Comment:
I like the idea of removing USE_IMPORT_FOUNDATION define (I generally don't like the flag). But in the case we excluding the case when we want to run both tests for SwiftFoundation and Foundation of SwiftXCTests on macOS.

@modocache @briancroom any thoughts about this?